torch_geometric/utils/subgraph.py

```diff
@@ -75,6 +71,69 @@ def subgraph(subset: Union[Tensor, List[int]], edge_index: Tensor,
         return edge_index, edge_attr
 
 
+def bipartite_subgraph(subset: Union[PairTensor, Tuple[List[int], List[int]]],
+                       edge_index: Tensor, edge_attr: Optional[Tensor] = None,
+                       relabel_nodes: bool = False,
+                       num_nodes: Tuple[int, int] = None,
+                       return_edge_mask: bool = False):
+    r"""Returns the induced subgraph of :obj:`(edge_index, edge_attr)`
+    containing the nodes in :obj:`subset`, for a bipartite graph.
+
+    Args:
+        subset (PairTensor or tuple([int],[int])): The nodes to keep.
+        edge_index (LongTensor): The edge indices.
+        edge_attr (Tensor, optional): Edge weights or multi-dimensional
+            edge features. (default: :obj:`None`)
+        relabel_nodes (bool, optional): If set to :obj:`True`, the resulting
+            :obj:`edge_index` will be relabeled to hold consecutive indices
+            starting from zero. (default: :obj:`False`)
+        num_nodes (tuple, optional): The number of nodes.
```

Comment:
```suggestion
        size (Tuple[int, int], optional): The size of the bipartite graph.
```
I think renaming that to `size` makes more sense.